tools/build-on-freebsd

```diff
@@ -1,7 +1,9 @@
 #!/bin/sh
-# Since there is no official FreeBSD port yet, we need some way of building and
-# installing cloud-init. This script takes care of building and installing. It
-# will optionally make a first run at the end.
+# The official to install cloud-init on FreeBSD is via the port or the package
```

Comment:
```suggestion
# The official way to install cloud-init on FreeBSD is via the port or the package
```

Reply:
indeed, adding way breaks the entire 79 char limit, so i… added more words.